sdk/eidos_sdk/cpu/conversational_agent_cpu.py

```diff
@@ -17,10 +18,11 @@
     StopReason,
     LLMToolCallRequestEvent,
     ToolCallStartEvent,
-    ToolCallEndEvent,
+    EndStreamContextEvent,
```

Comment:
I have the need for addition context demarcators for my multi-agent conversation stuff. So I created a sub-class of this. Can you put back the ToolCallEndEvent as I need to subclass the other.